.github/workflows/hrm-lambda-deploy.yml

```diff
@@ -35,8 +35,14 @@ on:
           - contact-retrieve-recording-url
           - contact-retrieve-transcript
           - resources-import-consumer
+          - resources-import-producer
           - resources-search-complete
           - resources-search-index
+      lambda_instance_id:
```

Comment:
This is kinda 🤮 but it works for now. Probably need to do something to make the producer deploy all the producer lambdas for the selected environments going forward